src/utils.js

```diff
@@ -0,0 +1,253 @@
+import { makeStyles } from "tss-react/mui";
+import { v4 as uuidv4 } from "uuid";
+
+const styles = makeStyles()(() => {
+  return {
+    textFieldStyle: {
+      marginBottom: 14,
+      width: "80%",
+    },
+  };
+});
+
+const getFromLocalStorage = () => {
+  const dataUser = JSON.parse(localStorage.getItem("userInformation"));
+  return dataUser;
+};
+
+const saveUserInformation = () => {
+  let userInformation = null;
+
+  userInformation = {
+    name: "Yesenia",
+    lastName: "Gonzalez",
+    email: "g@gmail.com",
+    password: "dilan123",
+    isLoggedIn: true,
+    idUser: uuidv4(),
+  };
+
+  localStorage.setItem("userInformation", JSON.stringify(userInformation));
+};
+
+/* saveUserInformation(); */
+
+const getEventsFromLocalStorage = (pageNumber) => {
+  const userInformation = getFromLocalStorage();
+  const userId = userInformation.idUser;
+  const events = JSON.parse(localStorage.getItem("Events"));
+
+  const eventsPerPage = 6;
```

Comment:
let's create another function called, `filterEventsByUserId`, inside this function you can have the code from line 40 to 75, and return the events filtered, call the new function `filterEventsByUserId` inside `getEventsFromLocalStorage`

Reply:
I used this getEventsFromLocalStorage() function to make this